.github/workflows/release.yml

```
@@ -0,0 +1,222 @@
+      - uses: actions/attest-build-provenance@v2
+        with:
+          subject-path: dist/*
+
+      - name: Publish GitHub Release
+        if: github.ref_type == 'tag' || (github.event_name == 'workflow_dispatch' && inputs.tag != '')
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: ${{ inputs.tag || github.ref_name }}
+          generate_release_notes: true
+          fail_on_unmatched_files: true
+          overwrite: true
```

actionlint (1.7.12)

[error] input "overwrite" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "overwrite_files", "prerelease", "preserve_order", "previous_tag", "repository", "tag_name", "target_commitish", "token", "working_directory"

(action)